front/src/components/comments/CommentChip.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled';
+import { FaRegComment } from 'react-icons/fa';
+
+export type CommentChipProps = {
+  count: number;
+  onClick?: () => void;
+};
+
+const StyledChip = styled.div`
+  height: 26px;
+  min-width: 34px;
+
+  padding-left: 2px;
+  padding-right: 2px;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  gap: 2px;
+
+  background: ${(props) => props.theme.secondaryBackgroundTransparent};
+  backdrop-filter: blur(6px);
```

Comment:
I'm not sure this backdrop filter will work given that the cell already has a backdrop filter.
In anycase, you should use overlayBackground from themes.ts